kornia/geometry/nerf/nerfmm/models/poses.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import torch.nn as nn
+
+from kornia.geometry.nerf.nerfmm.utils.lie_group_helper import make_c2w
+
+
+class LearnPose(nn.Module):
```

Comment:
similar to `PinholeCamera` this could be a class `PinholeCameraExtrinsics` or `PinholeCameraPose` with learnable parameters.